.github/environment.yml

```diff
@@ -0,0 +1,12 @@
+name: zchunk_test_env
+channels:
+- conda-forge
+dependencies:
+- libcurl
+- zstd
+- meson
+- c-compiler
+- ninja
+- pkg-config
+- cmake
```

Comment:
I don't want this included in Linux (and, if it's not needed by MacOS, I'd prefer it not be included there)

Reply:
OK, we can remove it